doc/api/async_hooks.md

```diff
@@ -107,7 +107,7 @@ specifics of all functions that can be passed to `callbacks` is in the
 const async_hooks = require('async_hooks');
 
 const asyncHook = async_hooks.createHook({
-  init(asyncId, type, triggerAsyncId, resource) { },
+  init(asyncId, type, triggerAsyncId, resource, bootstrap) { },
```

Comment:
The other way we could do this would be to use a special trigger ID for bootstrap

Reply:
By special ID do you mean something like making it `-1` for the bootstrap?

Reply:
Yeah, something along those lines. Rather than negative tho, since Node.js is the one assigning the numbers, we could just say that a `triggerAsyncId` of `1` === bootstrap.